example.php

```diff
@@ -54,38 +57,38 @@
 	"description" => "Guitarra"
 );
 
-$updated_product = $facturapi->Products->update("5a3f3e35f508333611ad6b3e",$product);
-var_dump($updated_product);
+$updated_product = $facturapi->Products->update( "5a3f3e35f508333611ad6b3e", $product );
+var_dump( $updated_product );
 
 $facturapi->Products->delete( "5a3f3e35f508333611ad6b3e" );
 
 var_dump( $facturapi->Customers->all() );
 
 
-var_dump( $facturapi->Customers->retrieve("5a3ee743f508333611ad6b3c") );
+var_dump( $facturapi->Customers->retrieve( "5a3ee743f508333611ad6b3c" ) );
 
 
 $customer = array(
-	"email" => "test@test.com",
+	"email"      => "test@test.com",
 	"legal_name" => "Testa Mesta",
-	"tax_id" => "RFC",
-	"address" => array(
-		"zip"=> "44940",
+	"tax_id"     => "RFC",
+	"address"    => array(
+		"zip"    => "44940",
```

Comment:
This lines have different indentation. Let's stick to using spaces instead of tabs to avoid this kind of problems.